nexus_constructor/off_renderer.py

```diff
@@ -7,11 +7,14 @@
 """
 
 from nexus_constructor.pixel_data import PixelData, PixelGrid
-from nexus_constructor.geometry import OFFGeometry
 from PySide2.Qt3DRender import Qt3DRender
 from PySide2.QtGui import QVector3D
 import struct
 import itertools
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:  # pragma: no cover
+    from nexus_constructor.geometry.off_geometry import OFFGeometry
```

Comment:
As you've changed to strings for the OFFGeometry type hint, surely this import doesn't do anything?

Reply:
this is to suppress flake8 warnings, but shouldn't need to exist if i make them inherit from a base class